src/aioamqp_consumer_best/_connection_provider.py

```diff
@@ -0,0 +1,25 @@
+import abc
+from typing import Iterable
+
+from itertools import cycle
+
+from aioamqp_consumer_best.records import ConnectionParams
+
+
+class ConnectionProviderABC(abc.ABC):
```

Comment:
Maybe rename to something with `Policy` / `Strategy`, for example `ServerSelectionPolicy`?

Reply:
Maybe just using `LoadBalancingPolicy` would be better

Reply:
Yes, it's ok

Reply:
Done